koopovereenkomst-v3-simple/pages/steps_verkoper/Step3.tsx

```diff
@@ -3,19 +3,18 @@ import Typography from "@mui/material/Typography";
 import Stack from "@mui/material/Stack";
 import Button from "@mui/material/Button";
 import { Box } from "@mui/system";
-import VC, { VCType } from '../../src/VC';
-import { useState } from "react";
+import VC, { SolidVC, VCType } from '../../src/VC';
+import { useCallback, useState } from "react";
 
 import Image from "../../src/Image";
 import KadasterKnowledgeGraph from "../../src/KadasterKnowledgeGraph";
 
 export default function Step3({ step = 3, handleNext, handleBack = () => { } }) {
-  const [loadedBRKVC, setLoadedBRKVC] = useState({} as any);
+  const [loadedBRKVC, setLoadedBRKVC] = useState([] as any);
 
-  const handleVC = (vc) => {
-    // get a trigger from <VC> to enable the "Doorgaan" button
-    setLoadedBRKVC(vc);
-  };
+  const updateVCs = useCallback(async (vcs: SolidVC[]) => {
+    setLoadedBRKVC(vcs);
+  }, []);
```

Comment:
Zou tussen de blokhaken de methods moeten worden opgesomd die binnen deze callback gebruikt worden en mogelijk state changes tot gevolg hebben? Oftewel, zou deze niet moeten eindigen met:

```javascript
  }, [setLoadedBRKVC]);
```

Reply:
In principe wel ja. `setLoadedBRKVC` komt echter voort uit `useState()`, dan is dat niet nodig. Volg hierin vooral de aanbevelingen van m'n IDE, die geeft het netjes aan wanneer ik dit wel en niet moet doen 😉